package.json

```diff
@@ -59,6 +59,7 @@
     "ember-cli-htmlbars": "^1.1.1",
     "ember-cli-htmlbars-inline-precompile": "^0.3.3",
     "ember-truth-helpers": "1.2.0",
+    "fixtable": "^2.0.1",
```

Comment:
Would this be a good time to bump to fixtable 2.0.2? It looks like Mike patched a few days ago.